src/cf.data.pre

```diff
@@ -4697,13 +4697,16 @@ DOC_START
 	    RESPONSE
 
 		[http::]<Hs	HTTP status code received from the next hop
-		[http::]>Hs	HTTP status code sent to the client
+		[http::]>Hs	HTTP status code of the last sent reply.
 
-		[http::]<h	Reply header. Optional header name argument
-				as for >h
+		[http::]<h	The header of the last sent reply. If that last
```

Comment:
I would remove the `If...` sentence. The [terminology](https://wiki.squid-cache.org/ResponseTerms) already covers this nuance (and probably does an overall better job of doing it).